contrib/make_packages

```diff
@@ -5,8 +5,8 @@ contrib=$(dirname "$0")
 whereis pip3
 if [ $? -ne 0 ] ; then echo "Install pip3" ; exit ; fi
 
-rm $contrib/packages/ -r
```

Comment:
This was there to allow running the script from somewhere else.

Reply:
Could you explain where exactly? Have doubts about this change, but there seems to be no alternative other than running contrib/make_packages on project root directory.

Reply:
Anywhere, I guess. `$contrib` is set to the absolute path of the contrib directory. This enables you to call `make_packages` from e.g. `/root/` without having `/root/packages` removed.

Reply:
Not saying that this is strictly necessary, but if you remove it you should give your reason to do so and remove the line where that variable is set above.

Reply:
Sorry, brain fart. Meant ./electron-cash which seems looking for ./packages/ in the root project directory.
https://github.com/fyookball/electrum/blob/200ccfaad1dd87f58ec7aec2fec6b43c839765ed/electron-cash#L52-L53

Reply:
$contrib is still needed for https://github.com/agilewalker/electrum/blob/e5103a235cbfa868b6c500c78696eb54c0e9d93d/contrib/make_packages#L11

Reply:
Okay, that makes sense :+1: Thanks for clarifying.